sympy/polys/domains/tests/test_domains.py

```diff
@@ -575,6 +575,8 @@ def test_Domain_convert():
     assert ZZ.convert(x - x) == 0
     assert ZZ.convert(x - x, R.to_domain()) == 0
 
+    assert ZZ[E].convert(exp(3))
```

Comment:
This should test the value returned. I guess this could be in its own test function like `test_exponential_domain()`. Something like:
```
K = ZZ[E]
eK = K.from_sympy(E)
assert K.from_sympy(exp(3)) == eK ** 3
assert K.convert(exp(3)) == eK ** 3
```

Reply:
ok I will make the function